Quran/ms.lproj/Readers.strings

```diff
@@ -0,0 +1,50 @@
+"qari_minshawi_murattal_gapless" = "Minshawi Murattal (gapless)";
```

Comment:
Should you translate `gapless`?

Reply:
I have updated the translations for `(gapless)`